packages/current_gitlab/current_gitlab.0.6/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "GitLab plugin for OCurrent"
+description: """\
+OCurrent provides an OCaml eDSL for writing CI/CD pipelines.
+
+This package provides primitives for interacting with GitLab.
+It can monitor and clone remote GitLab repositories, and can
+push GitLab status messages to show the results of testing
+PRs and branches."""
+maintainer: "timmcgil@gmail.com"
+authors: "timmcgil@gmail.com"
+homepage: "https://github.com/ocurrent/ocurrent"
+bug-reports: "https://github.com/ocurrent/ocurrent/issues"
+depends: [
+  "current" {= version}
+  "current_git" {= version}
+  "current_web" {= version}
+  "ocaml" {>= "4.08.0"}
+  "fmt" {>= "0.8.9"}
+  "lwt"
+  "yojson"
+  "cohttp-lwt-unix" {>= "4.0.0"}
+  "dune" {>= "2.9"}
+  "gitlab-unix" {= "0.1.2"}
```

Comment:
This seems rather strict. Any chance this could be unlocked to other minor releases?
```suggestion
  "gitlab-unix" {>= "0.1.2"}
```